Sources/BloomFilterWrapper/BloomFilterWrapper.swift

```diff
@@ -22,18 +22,22 @@ import Foundation
 public final class BloomFilterWrapper {
     private let bloomFilter: BloomFilterObjC
 
+    @inline(never)
```

Comment:
One thing I noticed in the crash logs is that these methods seem to be optimized marked as `compiler-generated`.  This normally means those methods have been optimized... possibly inlined.

Inlined methods are basically compiled in the locations where they're called, instead of kept as stand-alone methods.  I believe this may clash with `@_implementationOnly` since only this module has access to `BloomFilterObjC`, so an inlined method may fail to reference it.

By adding `@inline(never)` here I'm hoping to force these methods to be compiled as regular methods where they are.

Ref: https://embrace.io/blog/compiler-generated-ios-crashes/